lib/ChGovUk/Controllers/CustomerFeedback.pm

```diff
@@ -40,15 +40,18 @@ sub record_feedback {
     if (not $feedback->{customer_name}) {
         $feedback->{customer_name} = '(not provided)';
     }
-    
+
+    my $currentUrl = $feedback->{source_url};
+    my ( $previousUrl ) = $currentUrl=~ /sourceurl=([^&#]*)/;
+
     debug 'Writing customer FEEDBACK to ['.$self->ch_api->admin->customer_feedback->path.']';
 
     $self->ch_api->admin->customer_feedback->create({
         kind                => $feedback->{kind},
         customer_name       => $feedback->{customer_name},
         customer_email      => $feedback->{customer_email},
         customer_feedback   => $feedback->{customer_feedback},
-        source_url          => $feedback->{source_url},
+        source_url          => $previousUrl,
```

Comment:
the whole call to create could then be: `$self->ch_api->admin->customer_feedback->create($feedback);`